tools/extend/README.md

```diff
@@ -0,0 +1,42 @@
+# `check` CLI tool
+
+This binary is a thin wrapper around the `rtmr` library to
+extend the measurement into RTMR registers.
+
+The tool's input is the event log.
+
+The tool's output is an error or "Success".
+
+## Usage
+
+```
+./extend [options...]
+```
+
+### `-in`
```

Comment:
You have used a - here but not for the other flags. Please check.

Reply:
Oh, I followed the readme file from other documents such as https://github.com/google/go-tdx-guest/blob/main/tools/attest/README.md. Looks like we need to change them as well. Thanks for the finding.

Reply:
https://github.com/google/go-tdx-guest/pull/45 is created